src/app/core/services/integration/tracking.service.ts

```
@@ -64,9 +65,11 @@
       ...flattenedObject,
       Asset: 'Integration Settings Web'
     };
+    console.log('attempting:', action, properties);
```

ESLint

[error] Unexpected console statement.

(no-console)